os/arch/arm/src/armv7-r/arm_assert.c

```diff
@@ -972,5 +980,19 @@ void up_assert(const uint8_t *filename, int lineno)
 	(void)boardctl(BOARDIOC_RESET, 0);
 #endif
 
+#ifdef CONFIG_FAULT_MGR
+	fault_queue = mq_open("fault_queue", O_WRONLY | O_CREAT,  0666, 0);
+	if (fault_queue == (mqd_t)ERROR) {
+		svdbg("Can't open message queue\n");
```

Comment:
Else is needed for a failure of mq_open.

Reply:
Added the assert for the failure case.
Thanks